### ISSUE TITLE
Ramda

### ISSUE THREAD

**Issue by slavaGanzin** [casualjs/f#2]
Maybe better to use your time and move ramda to es6? It does everything you want to implement.

**Comment by aewens** [comment]
It really depends on the intention of the library. If he wants to make something that he intends on others using, then reimplementing Ramba would be a worthwhile idea. However, this code is complimented by a blog series which leads me to believe that it's intended as a self-made exercise to understand JavaScript better in the scope of functional programming while at the same time trying to make sense of the complexity of Haskell. If the latter is the case, this provides a better outlet of learning in the context of educating not only one's self but for others as well in how to approach functional JavaScript conceptually rather than dealing with the nitty-gritty quirks of JavaScript you encounter when trying to make something for everyone to use in production given the fragmentation in JavaScript engines across multiple browsers (I'm looking at you, IE).

tl;dr: Eh, maybe he should, but maybe he shouldn't.

**Comment by mateogianolio** [comment]
Could not have put it better myself @aewens.

**Comment by slavaGanzin** [comment]
@mateogianolio wish you a great progress
